pytorch_lightning/callbacks/base.py

```diff
@@ -178,3 +178,19 @@ def on_before_zero_grad(self, trainer, pl_module, optimizer):
         Called after optimizer.step() and before optimizer.zero_grad().
         """
         pass
+
+    def log(self, *_, **__):
+        """
+        This function is a placeholder. Don't override this function.
+        When using Lightning Trainer, the log function from
+        the model will be attached to this callback.
+        """
+        raise NotImplementedError
```

Comment:
```suggestion
        pass
```
Does this tell me as a user that I shall overwrite it instead of do not touch it?

shall we rather check (in the time of assignment log from model to callback) if the method log was overwritten and if so warn the user that his call is lost...

UPDATE: I see the overwritten check bellow

Reply:
Yes, I added the warning.